### ISSUE TITLE
USWDS - Bug: The Pagination buttons cut off at high zoom

### ISSUE THREAD

**Issue by babsdenney** [uswds/uswds#5981]
### Describe the bug

When viewing the [pagination](https://designsystem.digital.gov/components/pagination/) at the 320px viewport the left and right buttons are cutoff. 

<img width="515" alt="Screenshot 2024-07-16 at 12 59 39 PM" src="https://github.com/user-attachments/assets/1fde3726-a8ee-4819-b186-3475e8539476">

### Steps to reproduce the bug

View [pagination](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/?path=/story/components-pagination--default) at a small mobile width (320px).

### Expected Behavior

It's expected that the pagination is not cut off on the left and right sides and there is some space on the sides. By reducing the space around the ellipsis then the buttons can fit. This might be a solution, there are probably other ways to fix this issue as well. 

![image](https://github.com/user-attachments/assets/ac75a172-63e9-40c6-b5b7-9117289ff1d6)

### Related code

Additional code for overflow button on small screens
```css
.usa-pagination__overflow {
    margin-left: 0;
    margin-right: 0;
    min-width: 2rem;
}
```

### System setup

- USWDS Version: 3.8.1
- Mac OS 14.5 (23F79)
- Chrome Version 126.0.6478.127

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by amyleadem** [comment]
We should start by exploring possible solutions for this.

**Comment by amyleadem** [comment]
The engineering team has confirmed that these elements do get cut off at high zoom. Passing to the accessibility team to understand if this fails WCAG criteria and how common this pattern is. 

1. Which WCAG rule(s) apply to this issue? Does it pass or fail these rules? 
2. Can you help us assess the priority of this issue by letting us know if this is a common issue?

**Comment by brunerae** [comment]
@uswds/a11y - needs confirmation! Thank you!

**Comment by alex-hull** [comment]
1. Yes, I think would fail WCAG criteria [1.4.10, reflow](https://www.w3.org/WAI/WCAG22/Understanding/reflow.html). I think this could also potentially fail [2.5.8, target size](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html) as well. But it would most definitely fail the reflow criteria, as this is specific to mobile view and high levels of zoom. 
2. Since this does hinder the mobile user experience, and we want to create a fair experience for all, I would prioritize at a moderate to high level of need. @amyleadem

**Comment by dct2023** [comment]
@cathybaptista After touching base with @mejiaj it sounds like this is something we need to prioritize. Do you have the bandwidth to start on this one? If we can start on this during the sprint and it gets finished out at the beginning of the next sprint that would work.

**Comment by mejiaj** [comment]
@dct2023 we've re-assigned, but on further check-in we should bump this issue to `2025.3`.

**Comment by ryparker** [comment]
I reproduced the clipping on September 17, 2026, using `eaef39533a0a0f78976d6888edf8ae56fe9334d6`.

At a 320px CSS viewport, the default pagination runs from x=-8px to x=328px, clipping the outer edges of the first and last numbered controls. The Previous/Next arrow items are intentionally hidden at this width.

I haven't tested actual 400% browser zoom. A fix should cover both narrow viewports and browser zoom while keeping the controls easy to activate.

[Reproduction details and limitations](https://github.com/uswds/uswds/blob/c9a34f1ddc99b7d427db25ff06ab9913a699f627/reproductions/README.md#issue-5981) · [Screenshot](https://github.com/uswds/uswds/blob/c9a34f1ddc99b7d427db25ff06ab9913a699f627/reproductions/5981.png)
